content/references/data-api.md

```diff
@@ -1944,6 +1947,128 @@ Response:
 ```
 
 
+## Get External Markets
+[[Source]<br>](https://github.com/ripple/rippled-historical-database/blob/develop/api/routes/network/externalMarkets.js "Source")
+
+Get aggregated exchange volume from a list of off ledger exchanges for a specified rolling interval.
+
+The API returns results in units of a single _display currency_ rather than many different currencies. The conversion uses standard rates to and from XRP.
+
+#### Request Format ####
+
+<!-- MULTICODE_BLOCK_START -->
+
+*REST*
+
+```
+GET /v2/network/external_markets
+```
+
+<!-- MULTICODE_BLOCK_END -->
+
+[Try it! >](data-api-v2-tool.html#get-external-markets)
+
+Optionally, you can provide the following query parameters:
+
+| Field    | Value   | Description |
+|----------|---------|-------------|
+| `period` | String  | Aggregation Period - valid intervals are `1hour`, `1day`, `3day`, `7day`, or `30day`. Defaults to `1day`. |
+| `exchange_currency` | String - [Currency Code][] | Normalize all amounts to use this as a display currency. If not XRP, `exchange_issuer` is also required. Defaults to XRP. |
+| `exchange_issuer` | String - [Address][] | Normalize results to the specified `currency` issued by this issuer. |
+
+
+#### Response Format ####
+A successful response uses the HTTP code **200 OK** and has a JSON body with the following:
+
+| Field  | Value | Description |
+|--------|-------|-------------|
+| `result` | String | The value `success` indicates that this is a successful response. |
+| `data` | Object | Contains data for the specified period |
+| `data.date` | String | Date at which this period was calculated. |
+| `data.total` | Number | Total XRP volume exchanged during the period. |
+| `data.period` | String | Name of the period queried. |
+
+Each object in the `components` array of the Volume Objects represent the volume of a single external market.  Not all fields will be present for each market, depending on availability.
+
+| Field  | Value  | Description |
+|--------|--------|-------------|
+| `source` | String | Domain name of the specific external market. |
+| `base_volume` | Number | Exchange volume in terms of the base currency (XRP) |
```

Comment:
Will this always be XRP? The meaning of the parens here isn't clear to me. Also, I think the description is missing a period (.).

Reply:
nitpick: Hmm, looks like all of the descriptions are inconsistent w/r/t to periods. We should probably add to all or remove from all.

Reply:
I didn't write this one, but I suspect that this means the base currency is always XRP.

As for the periods, my latest commit should fix that.